File: C/URI/1040.c
#include <stdio.h>

void main(){
	float n1,n2,n3,n4,media,ne,mf;
	
	scanf("%f%f%f%f",&n1,&n2,&n3,&n4);
	media = (2*n1+3*n2+4*n3+n4)/10;
	printf("Media: %.1f\n", media);
	if (media >=7)
		printf("Aluno aprovado.\n");
	else if (media<5)
		printf("Aluno reprovado.\n");
	else if (media>=5 && media<7){
		printf("Aluno em exame.\n");
		scanf("%f",&ne);
		mf = (media+ne)/2;
		printf("Nota do exame: %.1f\n", ne);
		if (mf>=5){
			printf("Aluno aprovado.\n");
			printf("Media final: %.1f\n", mf);
		}
		else{
			printf("Aluno reprovado.\n");
			printf("Media final: %.1f\n", mf);
		}
		
	}
}

File: python/URI/1040.py
n1,n2,n3,n4=input().split()
n1=float(n1)
n2=float(n2)
n3=float(n3)
n4=float(n4)
media = (n1*2+n2*3+n3*4+n4*1)/10
print("Media: {:.1f}".format(media))
if media>=7:
    print("Aluno aprovado.")
elif media<5:
    print("Aluno reprovado.")
elif media>=5 and media<7:
    print("Aluno em exame.")
    nexame=input()
    nexame=float(nexame)
    print("Nota do exame: {:.1f}".format(nexame))
    media=(media+nexame)/2
    if media>=5:
        print("Aluno aprovado.")
    else:
        print("Aluno reprovado.")
    print("Media final: {:.1f}".format(media))

